### DISCUSSION TITLE
formData Put request!!!!.

### DISCUSSION THREAD

**Discussion by augustineonu** [cfug/dio#2085] [Q&A]
Does Dio not support PUT request for formData ?
i can't successfully send a file path. as it keeps sending its instance.

**Comment by kuhnroyal** [comment]
Like it was said in the old issue, nobody can help you unless you post a new issue with a minimal example that reproduces your problem.
Also please follow some general practice of respectful communication. If you need help, adding 4 exclamation marks to a discussion or issue title is usually not a great way to get it.
